movies/views/celebrities/index.ejs

```diff
@@ -0,0 +1,15 @@
+<link rel='stylesheet' href='/stylesheets/style.css' />
+
+<h2>List of Celebs</h2>
+
+<ul>
+  <% celebrities.forEach((celeb) => { %>
+    <li>
+      <a href="/celebrities/<%= celeb.id %>">Name: <%= celeb.name %></a>
+      <form style="display: inline;" action="/celebrities/<%= celeb.id %>/delete" method="POST"><input type="submit"  value="Delete"></form>
+      <form style="display: inline;" action="/celebrities/<%= celeb.id %>/edit" method="GET"><input type="submit"  value="Edit"></form>
+    </li>
+
```

Comment:
This view is good, but it lacks a link so the user can access the celebrities/new view. In this case, I had to manually type the url in the browser to be able to add one. Think about a user accessing the page the first time and what he would want to do :)`

Reply:
Big oversight on my part. I'll definitely update that while doing the movies part of the exercise!